### PR TITLE
[TEST] Increase sleep time of the query test

### PR DESCRIPTION
**Pull request by gichan-jang**
1 second sleep time is not enough to operate query server.
see: https://github.com/nnstreamer/nnstreamer/issues/3657#issuecomment-1038708557

Signed-off-by: gichan <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: tests/nnstreamer_query/runTest.sh [hunk 1]
```diff
@@ -20,7 +20,8 @@ testInit $1
 
 PATH_TO_PLUGIN="../../build"
 PERFORMANCE=0
-TIMEOUT_SEC=5
+SLEEPTIME_SEC=5
```
Comment (kparichay) [inline tests/nnstreamer_query/runTest.sh:23]: I recommend to please add a comment here about setting this as 5 sec or link the issue number of future reference.

**Review by kparichay — APPROVED** [review]
LGTM, good find

File: tests/nnstreamer_query/runTest.sh (hunk 1 above)
Comment (gichan-jang) [reply tests/nnstreamer_query/runTest.sh:25]: I added the comment. Thanks.

**Review by myungjoo — APPROVED** [review]
LGTM

**Pull request merged**